projects/aca-content/src/lib/components/sidenav/user-menu/user-menu.component.ts

```diff
@@ -50,9 +50,28 @@ export class UserMenuComponent implements OnInit {
   @Input()
   data: { items: any[] };
 
+  @ViewChild(MatMenu)
+  menu: MatMenu;
+
+  @ViewChildren(ToolbarMenuItemComponent)
+  toolbarMenuItems: QueryList<ToolbarMenuItemComponent>;
+
   ngOnInit() {
     if (this.data?.items) {
       this.data.items.sort((a, b) => a.order - b.order);
     }
   }
+
+  ngAfterViewInit() {
+    const menuItems: MatMenuItem[] = [];
+    this.toolbarMenuItems.forEach((toolbarMenuItem: ToolbarMenuItemComponent) => {
```

Comment:
same comment like in ADF - we can replace forEach with filter which will cause a shorter code: 
https://github.com/Alfresco/alfresco-ng2-components/pull/10647#discussion_r1956054188

Reply:
done